modules/ROOT/pages/contribute.adoc

```diff
@@ -7,15 +7,12 @@ Many of our documentation repositories are open and we welcome your contribution
 
 To contribute to the English version of this documentation:
 
-. Select *Edit in Github* on the page you want to modify.
+. If this is your first time contributing, sign the https://cla.salesforce.com/sign-cla[Salesforce.com Contributor License Agreement (CLA)]. The CLA only needs to be signed once per GitHub account.
```

Comment:
Placement of "only"; see associated entry in https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+Writing+Style+Reference.

```suggestion
. If this is your first time contributing, sign the https://cla.salesforce.com/sign-cla[Salesforce.com Contributor License Agreement (CLA)]. The CLA needs to be signed only once per GitHub account.
```